src/test/java/com/oltpbenchmark/api/TestConfigurationLoad.java

```diff
@@ -0,0 +1,270 @@
+                            <statement name="getBuyerInfo">SELECT u_id, u_balance FROM useracct WHERE u_id = ?</statement>
+                            <statement name="getItemInfo">SELECT i_num_bids, i_current_price, i_end_date, ib_id, ib_buyer_id, u_balance FROM item, item_max_bid, item_bid, useracct WHERE i_id = ? AND i_u_id = ? AND imb_i_id = i_id AND imb_u_id = i_u_id AND imb_ib_id = ib_id AND imb_ib_i_id = ib_i_id AND imb_ib_u_id = ib_u_id AND ib_buyer_id = u_id</statement>
+                            <statement name="updateUserBalance">UPDATE useracct SET u_balance = u_balance + ? WHERE u_id = ?</statement>
+                            <statement name="insertUserItem">INSERT INTO useracct_item (ui_u_id, ui_i_id, ui_i_u_id, ui_ip_id, ui_ip_ib_id, ui_ip_ib_i_id, ui_ip_ib_u_id, ui_created) VALUES (?, ?, ?, ?, ?, ?, ?, ?)</statement>
+                            <statement name="updateUserItem">UPDATE useracct_item SET ui_ip_id = ?, ui_ip_ib_id = ?, ui_ip_ib_i_id = ?, ui_ip_ib_u_id = ? WHERE ui_u_id = ? AND ui_i_id = ? AND ui_i_u_id = ?</statement>
+                            <statement name="updateItem">UPDATE item SET i_status = 3, i_updated = ? WHERE i_id = ? AND i_u_id = ?</statement>
+                        </statements>
+                    </procedure>
+                    <procedure procedureClass="com.oltpbenchmark.benchmarks.auctionmark.procedures.ResetDatabase">
+                        <statements>
+                            <statement name="deleteItemPurchases">DELETE FROM item_purchase WHERE ip_date &gt; ?</statement>
+                            <statement name="resetItems">UPDATE item SET i_status = ?, i_updated = ? WHERE i_status != ? AND i_updated &gt; ?</statement>
+                            <statement name="getLoaderStop">SELECT cfp_loader_stop FROM config_profile</statement>
+                        </statements>
+                    </procedure>
+                    <procedure procedureClass="com.oltpbenchmark.benchmarks.auctionmark.procedures.UpdateItem">
+                        <statements>
+                            <statement name="getMaxItemAttributeId">SELECT MAX(ia_id) FROM item_attribute WHERE ia_i_id = ? AND ia_u_id = ?</statement>
+                            <statement name="insertItemAttribute">INSERT INTO item_attribute(ia_id,ia_i_id,ia_u_id,ia_gav_id,ia_gag_id) VALUES (?, ?, ?, ?, ?)</statement>
+                            <statement name="deleteItemAttribute">DELETE FROM item_attribute WHERE ia_id = ? AND ia_i_id = ? AND ia_u_id = ?</statement>
+                            <statement name="updateItem">UPDATE item SET i_description = ?, i_updated = ? WHERE i_id = ? AND i_u_id = ?</statement>
+                        </statements>
+                    </procedure>
+                </procedures>
+            </dialect>
+            """;
+
+    public void testLoadDatabase() throws JsonProcessingException {
```

Comment:
If I'm reading this file correctly, all the tests just print the final string at the end. So they're useful if ran and inspected visually, but not useful for CI "did something break" purposes.

Thoughts on saving the string that is currently output and comparing against it to make sure it doesn't change? Admittedly, that would also be a very brittle test. Just thinking out loud

Reply:
Good comment and suggestion.  I'll update the test.  I did this quickly as a way to manually check my work but I can make it more robust.